spec/javascripts/components/story/expanded_story/expanded_story_estimate_spec.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { ExpandedStoryEstimate } from 'components/story/ExpandedStory/ExpandedStoryEstimate';
+
+describe('<ExpandedStoryEstimate />', () => {
+  it("renders component with 'Fibonacci' point scale in select", () => {
+    const project = { pointValues: ['1','2','3','5','8'] };
+    const story = { estimate: null };
+
+    const wrapper = shallow(<ExpandedStoryEstimate project={project} story={story}/>);
+    const select = wrapper.find('select').text();
+
+    project.pointValues.forEach((value) => {
+      expect(select).toContain(value);
+    });
+  });
+
+  it("renders component with 'Powers of two' point scale in select", () => {
+    const project = { pointValues: ['1','2','4','8'] };
+    const story = { estimate: null };
+
+    const wrapper = shallow(<ExpandedStoryEstimate project={project} story={story}/>);
+    const select = wrapper.find('select').text();
+
+    project.pointValues.forEach((value) => {
+      expect(select).toContain(value);
+    });
+  });
+
+  it("renders component with 'Linear' point scale in select", () => {
+    const project = { pointValues: ['1','2','3','4','5'] };
+    const story = { estimate: null };
+
+    const wrapper = shallow(<ExpandedStoryEstimate project={project} story={story}/>);
+    const select = wrapper.find('select').text();
+
+    project.pointValues.forEach((value) => {
+      expect(select).toContain(value);
+    });
+  });
+
+  describe("When story.estimate is not null", () => {
+    it("sets the select defaultValue as story.estimate", () => {
```

Comment:
It would be better to use multiple `it`s here (you can loop through an array and call `it` inside of it) instead of rendering the same component multiple times inside a single `it`